selfdrive/test/process_replay/model_replay.py

```diff
@@ -207,10 +207,12 @@ def do_GET(self):
         'logMonoTime',
         'modelV2.frameDropPerc',
         'modelV2.modelExecutionTime',
+        'modelV2.locationMonoTime',
         'driverStateV2.modelExecutionTime',
         'driverStateV2.dspExecutionTime',
         'navModel.dspExecutionTime',
         'navModel.modelExecutionTime',
+        'navModel.locationMonoTime',
```

Comment:
why isn't this deterministic? wouldn't the replay outputs differ if it wasn't?

Reply:
I believe they are deterministic, they just require the refs to be updated and I wanted to keep them the same. https://jenkins.comma.life/blue/organizations/jenkins/openpilot/detail/fix-nav-flicker/2/pipeline/70